src/lib/Util.ts

```diff
@@ -124,6 +124,28 @@ export class Util {
         }
     }
 
+    public static sanitizeForInsert (arr:any, sObjectName:string)  {
+      
+        for (let i = 0; i < arr.length; i++) {
+            for (let prop in arr[i]) {
+                if (prop === 'attributes') delete arr[i][prop];    
+                if ((prop ==='Pricebook2' || prop ==='Product2') && arr[i][prop] == null){
+                    arr[i][prop] ={}; 
+                    arr[i][prop]['enxCPQ__TECH_External_Id__c'] = null;
```

Comment:
Just to be sure - is null required since you've created empty object on that property?